src/cmd/flux-job.c

```diff
@@ -93,8 +93,8 @@ static struct optparse_option list_inactive_opts[] =  {
     { .name = "count", .key = 'c', .has_arg = 1, .arginfo = "N",
       .usage = "Limit output to N jobs",
     },
-    { .name = "timestamp", .key = 't', .has_arg = 1, .arginfo = "T",
-      .usage = "Limit output to jobs newer than timestamp",
+    { .name = "since", .key = 's', .has_arg = 1, .arginfo = "T",
+      .usage = "Limit output to jobs newer than since",
```

Comment:
Suggested change for help string: `Limit output to jobs that entered the inactive state since timestamp T`